### PR TITLE
Addwatermark

### PR DESCRIPTION
**Pull request by jonathanarde**
add watermark text 'Powered by Glorify.cc' at the bottom center of pptx slide

### CODE REVIEW

File: routes/user.playlist.js [hunk 1]
```diff
@@ -227,7 +227,7 @@ router.route('/:playlist_name/export3')
         songs2d = [];
         maxNumberOfSongs = 0;
         Playlist.find({
-                owner: req.user._id,
+                // owner: req.user._id,
```
Comment (chrislim) [inline routes/user.playlist.js]: still need this?

File: routes/user.playlist.js [hunk 2]
```diff
@@ -387,6 +400,18 @@ function generateSlideByStanza(res, songs2d, playlistName, stanzasPerSlide) {
                 }
                 margin += 150
             }
+            pObj = slide.addText('Powered by Glorify.cc', {
+                x: 'c', //x position
+                y: 620, //y position
+                cx: '100%', //width
+                cy: 20,
+                font_size: 30,
```
Comment (chrislim) [inline routes/user.playlist.js]: Is this still used for a different place?

File: routes/user.playlist.js [hunk 3]
```diff
@@ -360,6 +360,19 @@ function generateSlideByStanza(res, songs2d, playlistName, stanzasPerSlide) {
             });
             titleMargin += 100;
         })
+        pObj = slide.addText('Powered by Glorify.cc', {
```
Comment (chrislim) [inline routes/user.playlist.js]: Pull out the text into a variable shared between this and the next section of code that uses it.

**Comment by chrislim** [comment]
Will the handout support the watermark too?

**Comment by jonathanarde** [comment]
no it wont for now. we need to work on the pdf layout to limit how many stanzas it can display per page. therefore, we can prevent it to cut the stanzas between pages and make a room for watermark. it needs more work. I put the task under feature backlog because we dont really need it for christmast event

File: routes/user.playlist.js (hunk 2 above)
Comment (jonathanarde) [inline routes/user.playlist.js]: we will use it eventually when we implement line per slide

**Pull request merged**